openmdao/components/tests/test_ks_comp.py

```diff
@@ -121,6 +121,42 @@ def test_beam_stress(self):
         self.assertTrue(np.all(stress0 < 100.0))
```

Comment:
I think you need to add an example to the feature doc for KSComp.  This is mostly to address the weakness in the current documentation that led to this story, namely that none of the examples clue the user in to the fact that they need to add a constraint (or now, set the new option to True) in order for their KSComp to be active in the optimization problem.

Some of the existing docs and writing should also be updated because of this.

Reply:
Example added for `add_constraint=True`